tests/core/conftest.py

```diff
@@ -242,10 +242,23 @@ def trained_model(loop, project) -> Text:
 
 
 @pytest.fixture
-async def restaurantbot(trained_async, tmpdir_factory) -> Text:
+async def restaurantbot(trained_async) -> Text:
```

Comment:
We could add the type of `trained_async` here, but I'm not sure what it is (a callable since it is a function?)

Reply:
yeah it's all over the place, that's why I thought it might be better to do it in another pr to not mess this one up too much

Reply:
No problem!